lms/djangoapps/courseware/access.py

```diff
@@ -646,17 +646,18 @@ def _has_staff_access_to_descriptor(user, descriptor, course_key):
     return _has_staff_access_to_location(user, descriptor.location, course_key)
 
 
-def is_mobile_available_for_user(user, course):
+def is_mobile_available_for_user(user, descriptor):
     """
     Returns whether the given course is mobile_available for the given user.
     Checks:
         mobile_available flag on the course
         Beta User and staff access overrides the mobile_available flag
+    descriptor (CourseDescriptor|CourseOverview): course or overview of course in question
```

Comment:
To be consistent with our docstring convention, please add this under an `Arguments:` section.